app/src/main/java/org/kiwix/kiwixmobile/main/AddNoteDialog.java

```diff
@@ -106,15 +102,19 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
       ZIM_NOTES_DIRECTORY = NOTES_DIRECTORY + zimNoteDirectoryName + "/";
     } else {
-      showToast(R.string.error_filenotfound, Toast.LENGTH_LONG);
-      closeKeyboard();
-      getFragmentManager().beginTransaction().remove(AddNoteDialog.this).commit();
+      onFailureToCreateAddNoteDialog();
     }
   }
 
+  private void onFailureToCreateAddNoteDialog() {
+    showToast(R.string.error_filenotfound, Toast.LENGTH_LONG);
+    closeKeyboard();
+    getFragmentManager().beginTransaction().remove(AddNoteDialog.this).commit();
```

Comment:
you don't need to qualify `this` with `AddNoteDIalog`

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1387/commits/06fcfa55d5643c6baf0204422505fb087db95cb6#diff-36e13fa4a9132616aee81e1f88bfa0ffR112)